db/migrate/20230616152325_add_post_ref_to_like.rb

```diff
@@ -0,0 +1,5 @@
+class AddPostRefToLike < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :post, null: false, foreign_key: true
+  end
+end
```

Comment:
- [ ][optional]  @muhammedb8826 kindly remove this migration file, it duplicates a similar file `20230616151743_add_user_ref_to_like.rb` making your migration fail to run , you get the error below 20230616151743_add_user_ref_to_like.rb